package.default.json

```diff
@@ -59,8 +59,8 @@
     "nconf": "^0.8.5",
     "nodebb-plugin-composer-default": "6.0.5",
     "nodebb-plugin-dbsearch": "2.0.9",
-    "nodebb-plugin-emoji-extended": "1.1.1",
-    "nodebb-plugin-emoji-one": "1.2.1",
+    "nodebb-plugin-emoji": "2.0.0",
+    "nodebb-plugin-emoji-one": "2.0.0",
```

Comment:
I believe this upgrade path (as discussed) would leave existing users of emoji-extended with broken emoji sets, since emoji-extended does not work with v2 of the emoji packs...